src/smc_pyutil/smc_pyutil/start_smc.py

```diff
@@ -14,14 +15,21 @@
 def cmd(s):
     print s
     if os.system(s):
-       sys.exit(1)
+        sys.exit(1)
 
 def started():
     return os.path.exists("%s/local_hub/local_hub.port"%SMC)
 
 def main():
+    # optionally, tcp port and raw port numbers
+    port_args = ''
+    if len(sys.argv) >= 3:
+        port_args += ' %s' % sys.argv[2]
+    if len(sys.argv) >= 4:
+        port_args += ' %s' % sys.argv[3]
+
     # Start local hub server
-    cmd("smc-local-hub start")
+    cmd("smc-local-hub start" + port_args)
```

Comment:
Delete all the above added code and just do `cmd("smc-local-hub start " + ' '.join(port_args))`